babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I tried to let it into package.json, but I have no luck with that.
I hope jest will move to babel 7 soon ...